### PR TITLE
Create Privet-translation

### PR DESCRIPTION
**Pull request by orestotel**
Create translation
#1 #2

### CODE REVIEW

File: Privet-translation [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Eng - "Helop"
+Ru - "Privet"
```
Comment (orestotel) [inline Privet-translation]: I've done effort to complete this

File: Privet-translation [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Eng - "Helop"
```
Comment (IhorMruchko) [inline Privet-translation]: I`m not HOLOP, u`ve made a mistake

Comment (orestotel) [reply Privet-translation]: Changes are already being done

**Comment by orestotel** [comment]
close due to #3 #1

**Pull request closed without merging**